### PR TITLE
[Blueprints] Add published blueprint object to the payload to support change notes

### PR DESCRIPTION
**Pull request by filizt**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by erich-wang — CHANGES_REQUESTED** [review]
@filizt , please make sure all checks are passed, thanks.

**Comment by filizt** [comment]
> @filizt , please make sure all checks are passed, thanks.

The model validation failures in the checks are caused by a tooling issue here: https://github.com/Azure/oav/issues/431. We had an another PR earlier this week that had the same failures but the they were suppressed and PR was merged since the failures were also due to the tooling issue. See here: https://github.com/Azure/azure-rest-api-specs/pull/6297

Could you help suppress the failures and merge the PR?

File: specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintDefinition.json [hunk 1]
```diff
@@ -416,6 +416,15 @@
...
+          },
+          {
+            "name": "publishedBlueprint",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/PublishedBlueprint"
+            },
+            "description": "Published Blueprint to create or update."
           }
```
Comment (majastrz) [inline specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintDefinition.json]: Note for other reviewers:
The blueprint backend has always supported this and it was always optional. So SDKs generated from swaggers with and without this change will work just fine.

**Review by majastrz — APPROVED** [review]
Looks good from ARM side.

**Comment by filizt** [comment]
@erich-wang could you review the PR please?

**Review by erich-wang — APPROVED** [review]
approve the change since the check failure is caused by tool bug.

**Comment by erich-wang** [comment]
@sergey-shandar, the check failure is caused by a bug in ova, please merge this PR, thanks.

**Pull request merged**